Metric def for last commited tx id (#483)
neo4j/docs-aura: modules/ROOT/pages/platform/metrics-integration.adoc

| Default aggregation
m| MAX
|===

.Last Committed Transaction ID
[frame="topbot", stripes=odd, grid="cols", cols="<1,<4"]
|===
| Metric name
m| neo4j_database_transaction_last_committed_tx_id_total
| Description

> | The ID of the last committed transaction. Track this for primary & secondary members of your Aura instance. It might break into separate charts. It should show one line, ever-increasing, and if one of the lines levels off or falls behind, it is clear that this instance is no longer replicating data, and action is needed to rectify the situation. 

Reviewer: ```suggestion
| The id of the last committed transaction. 
Track this for primary and secondary instances of your Aura instance. 
It might break into separate charts. 
It shows one ever-increasing line and if one of the lines levels off or falls behind, it is clear that this instance is no longer replicating data, and action is needed to rectify the situation. 
```

| Default aggregation
m| MAX
|===

.Last Committed Transaction ID
[frame="topbot", stripes=odd, grid="cols", cols="<1,<4"]
|===
| Metric name
m| neo4j_database_transaction_last_committed_tx_id_total
| Description
| The id of the last committed transaction. 

> Track this for primary and secondary cluster members of your Aura instance. 

Author: ```suggestion
Track this for primary cluster members of your Aura instance. 
```

> It might break into separate charts. 

Author: ```suggestion
It should show overlapping, ever-increasing lines and if one of the lines levels off or falls behind, it is clear that this cluster member is no longer replicating data, and action is needed to rectify the situation.
```

> It shows one ever-increasing line and if one of the lines levels off or falls behind, it is clear that this cluster member is no longer replicating data, and action is needed to rectify the situation. 

Author: ```suggestion
```
